tests/providers/telegram/hooks/test_telegram.py

```diff
@@ -128,7 +128,7 @@ def test_should_send_message_if_chat_id_is_provided_through_constructor(self, mo
 
     @mock.patch("airflow.providers.telegram.hooks.telegram.TelegramHook.get_conn")
     def test_should_send_message_if_chat_id_is_provided_in_connection(self, mock_get_conn):
-        mock_get_conn.return_value = mock.Mock(password="some_token")
+        mock_get_conn.return_value = mock.AsyncMock(password="some_token")
```

Comment:
The problem more general, AsyncMock not existed in Python 3.7 and need to use library which provide backcompat, the way how it done in Google Provider

https://github.com/apache/airflow/blob/207f65b542a8aa212f04a9d252762643cfd67a74/tests/providers/google/cloud/utils/compat.py#L18-L37

You could also have a look this PR: https://github.com/apache/airflow/pull/28290

Reply:
Sorry, I should mention this before

Reply:
@Taragolis, yes I noticed, at some point today, that Python 3.7 did not contain `AsyncMock`. Should I reuse the implementation in Google Provider? I had just added [this](https://github.com/apache/airflow/pull/28953/files#diff-186f9ce31911e0f300377123cda4f4c7f310a3dbb48c59ce38f5171457022e8eR35). But reusing the implementation in Google Provider is probably a better idea.

Reply:
Doesn't actually matter if your solution work it is fine.
Anyway Python 3.7 almost [EOL](https://endoflife.date/python) and we will drop support 3.7 soon.